src/plugins/workspace/public/workspace_client.ts

```diff
@@ -176,16 +176,16 @@ export class WorkspaceClient {
    * @returns
    */
   public async create(
-    attributes: Omit<WorkspaceAttribute, 'id'> & {
-      permissions: WorkspaceRoutePermissionItem[];
-    }
+    attributes: Omit<WorkspaceAttribute, 'id'>,
+    permissions: WorkspaceRoutePermissionItem[]
```

Comment:
Maybe not in the scope, but should `permissions` be optional? Because user may not have permission control turned on.

Reply:
Sure, the permissions could be optional.